File: generator/output/ts-model/sms-log.model.ts
class SmsLogModel {
  public id: number = 0;

  // deleted 
  public deleted: boolean = false;

  // mobile 手机号
  public mobile: string;

  // code 验证码
  public code: string;

  public created_at: string;

  public updated_at: string;
}

const SmsLogModelFields = {
  id: "id",
  deleted: "deleted",
  mobile: "mobile",
  code: "code",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {SmsLogModel, SmsLogModelFields};

File: generator/output/entityModel/SmsLogModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Boolean;
import java.lang.Integer;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* SmsLog
*/
@Data()
public class SmsLogModel {
  public Long id = null;

  /**
  * 
  */
  public Boolean deleted = null;

  /**
  * 手机号
  */
  public String mobile = null;

  /**
  * 验证码
  */
  public Integer code = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}
